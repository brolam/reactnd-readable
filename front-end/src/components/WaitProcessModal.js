import React from 'react'
import PropTypes from 'prop-types'

function WaitProcessModal(props) {
  const { message } = props
  return (
    <div />
  )
}

WaitProcessModal.propTypes = {
  message: PropTypes.string.isRequired,
}

export default WaitProcessModal