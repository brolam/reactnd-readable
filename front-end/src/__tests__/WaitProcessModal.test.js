import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import { mount } from 'enzyme'
import WaitProcessModal from '../components/WaitProcessModal'

test('render without error', () => {
  ReactDOM.render(
    <WaitProcessModal
      message={"Please wait while the information is updated."}
    />, document.createElement('div'));
})

test('last Snapshot', () => {
  const waitProcessModal = renderer.create(
    <WaitProcessModal
      message={"Please wait while the information is updated."}
    />);
  let tree = waitProcessModal.toJSON();
  expect(tree).toMatchSnapshot();
})