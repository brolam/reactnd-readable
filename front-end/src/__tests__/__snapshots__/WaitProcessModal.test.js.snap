// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`last Snapshot 1`] = `
<div
  className="modal-short modal-open"
>
  <div
    className="modal-short-dialog wait-process-dialog"
  >
    <h1>
      Please wait while the information is updated.
    </h1>
  </div>
</div>
`;
